### PR TITLE
Ammended Mosaic Docs

### PR DESCRIPTION
**Pull request by PoisonPhang**
## Issue
N/A

## Description
Amended Mosaic docs with workflow and description.
Linked README.md in cargo docs.
Clarified cargo documentation

## Checklist

- [X] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [X] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Review by hussein-aitlahcen — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]

File: book/src/pallets/mosaic.md [hunk 1]
```diff
@@ -22,37 +20,41 @@ decay function decayer, which depends on the penalty, `current_block`, and
...
-The budget penalty decayer.
+The Mosaic pallet is comprised of three main components: the Relayer interface, 
+incoming transactions, and outgoing transactions.
 
-### NetworkId
+### The Relayer Interface
 
-Network identifier.
+The Relayer interface provides the necessary functionality for managing the 
+constrains of the Relayer and exposing the functionality needed by the Relayer
```
Comment (benluelo) [inline book/src/pallets/mosaic.md:31]: constraints
```suggestion
constraints of the Relayer and exposing the functionality needed by the Relayer 
```

File: frame/mosaic/README.md [hunk 2]
```diff
@@ -1,10 +1,60 @@
...
+The Mosaic Pallet implements the interface for the Mosaic Relayer. The Mosaic 
+Relayer will relay liqudity accross chains and layers.
+
+As opposed to the EVM-EVM bridge, this pallet takes a different approach and 
+uses mint and burn operations. Because of that it also limits the amount the 
+relayer can mint using a decaying penalty.
+
+### Decaying Penalty
+
+At moment N, the Relayer has a maximum budget budget. Minting a token adds a
```
Comment (benluelo) [inline frame/mosaic/README.md:16]: budget budget?

File: frame/mosaic/README.md [hunk 3]
```diff
@@ -1,10 +1,60 @@
...
+Relayer will relay liqudity accross chains and layers.
+
+As opposed to the EVM-EVM bridge, this pallet takes a different approach and 
+uses mint and burn operations. Because of that it also limits the amount the 
+relayer can mint using a decaying penalty.
+
+### Decaying Penalty
+
+At moment N, the Relayer has a maximum budget budget. Minting a token adds a 
+penalty penalty to the Relayer. The penalty decreases each block according to
```
Comment (benluelo) [inline frame/mosaic/README.md:17]: same here, maybe "penalty `penalty`" if you're referring to a variable of sorts?

Comment (PoisonPhang) [reply frame/mosaic/README.md]: Nope, just me being blind
32b481f76332c3f5a1fa991893aa48fa4193f22a

File: frame/mosaic/README.md (hunk 2 above)
Comment (PoisonPhang) [reply frame/mosaic/README.md]: 32b481f76332c3f5a1fa991893aa48fa4193f22a

File: book/src/pallets/mosaic.md (hunk 1 above)
Comment (PoisonPhang) [reply book/src/pallets/mosaic.md]: 32b481f76332c3f5a1fa991893aa48fa4193f22a

Comment (PoisonPhang) [reply book/src/pallets/mosaic.md]: Note, README and book/pallets synced here
617bec50d39cbbd1db640efacef83e65427980e1

File: book/src/pallets/mosaic.md [hunk 4]
```diff
@@ -2,57 +2,4 @@
...
-### ControlOrigin
-
-Origin capable of setting up the Relayer.
-
-Acts as a root or half council as they will be capable of stopping attacks.
-
-### WeightInfo
-
-Weight implementation used for extrinsics.
+{{#include ../../../frame/mosaic/README.md:5:}}
```
Comment (andor0) [inline book/src/pallets/mosaic.md:5]: looks like [this doesn't work](https://github.com/ComposableFi/composable/blob/connor/mosaic-docs/book/src/pallets/mosaic.md)

Comment (PoisonPhang) [reply book/src/pallets/mosaic.md:5]: This isn't for GitHub markdown. Its for the mdbook.
*note, the following expects that you have [mdbook](https://github.com/rust-lang/mdBook) installed*
```sh
cd book/
mdbook serve
```
and then visit http://localhost:3000/pallets/mosaic.html to view the changes before they are merged.

Comment (andor0) [reply book/src/pallets/mosaic.md:5]: ahh, I got it.
ok.

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by benluelo — APPROVED** [review]

**Pull request merged**